style.css

```diff
@@ -1 +1,193 @@
 /*  Add your styling here */
+
+body {
+    background-color: rgb(234, 239, 241);
+}
+
+
+.container {
+    display:grid;
+    grid-template-areas: "header  header  header  header  header  header  header   header  header  header  header  header"
+                         "welcome welcome welcome welcome welcome welcome welcome welcome welcome welcome welcome welcome"
+                         "text    text    text    text    text    text    text     text    text    text     text    text"
+                         "gallery gallery gallery gallery gallery gallery gallery gallery gallery gallery gallery gallery"
+                         "footer  footer  footer  footer  footer  footer  footer  footer  footer  footer  footer  footer";
+    grid-gap: 10px;
+    max-width: 100%;
+}
+
+header {
+    grid-area: header;
+    width: 100%;
+    height: 200px;
+    background-image: url("../HTML-CSS-Coursework-Week3/img/american-heritage-chocolate-vdx5hPQhXFk-unsplash.jpg");
+}
+
+
+.banner-text {
+    display: flex;
+    position: relative;
+    font-size: 16px;
+    width: 100%;
+    font-family: 'Estonia', cursive;
+    justify-content: center;
+}
+
+h1 {font-weight:1000;}
+
+
+.welcome {
+    grid-area: welcome;
+    max-width: 100%;
+}
+
+.main-text {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+}
+
+h2 {
+    font-family: 'Comforter', cursive;
+    font-size: 40px;
+    font-weight: 100;
+    margin-bottom: -20px;
+}
+
+p {
+    font-family: 'Estonia', cursive;
+    font-size: 40px;
+    font-weight: 1000;
+}
+
+.main-img {
+    width: 100%;
+    height: 200px;
+}
+
+
+.gallery {
+    grid-area: gallery;
+}
+
+.gallery-img {
+    width: 100%;
+    height: 150px;
```

Comment:
If you define an explicit `height` on your image, on screen resize, the image gets distorted.
One solutions here is this, because the grid will define the max size of each image. Otherwise, you need to define a wrapper around the image and give it explicit width and height.
```
.gallery-img {
   width: 100%;
   height: 100%;
   object-fit: cover;
}
```